assets/js/googlesitekit-data.js

```diff
@@ -1,10 +1,22 @@
 /**
  * External dependencies
  */
-import Data from 'assets/js/googlesitekit/data';
+
+/**
+ * Internal dependencies
+ */
+import SiteKitRegistry from 'assets/js/googlesitekit/data';
+import { collect, collectReducers } from 'assets/js/googlesitekit/data/utils';
```

Comment:
This file shouldn't have two import statements, these utility functions should be exposed as part of the `googlesitekit/data` module itself. The current behavior you have here leads to some uncertainty because if you want to use e.g. `collect`, it would work both by importing from `googlesitekit/data/utils` and from `googlesitekit/data`, since the latter would result in `window.googlesitekit.data` in the build which has it.

This ties in with the above in `googlesitekit-api.js`, we need to ensure that these wrapper files export the same functions on the global like the underlying module does.